test.sh

```diff
@@ -0,0 +1,3 @@
+./configure
```

Comment:
I suspect you didn't mean to check this in :)

Reply:
Woooops! :upside_down_face: